### ISSUE TITLE
Maximum password length

### ISSUE THREAD

**Issue by Amerlander** [appwrite/appwrite#1639]
## 🐛 Bug Report

Password requirements for root user is not clear / maximum password length

## Have you spent some time to check if this issue has been raised before?
yes

## To Reproduce

1) Setup Appwrite
2) Open Appwrite Webseite
4) Chose a password with more than 32 Characters

## Expected behavior

a) Get an error, that the password is too long
b) Accept longer passwords (preffered)

I'm not so much into password security, but I realy like beeing able to choose passwords without any length limitation. I know, that some technologies like bcrypt as e.g. have a length limit, but I know others are working arround by doing some hashing before. (like Dropbos does SHA-512 and then bcrypt on that hased string: https://dropbox.tech/security/how-dropbox-securely-stores-your-passwords)

## Actual Behavior

Get an error, that it doesn't work, and you should try again later

```
appwrite                        | [Error] Timestamp: 2021-09-19T15:12:50+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /v1/account
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: User (role: guest) missing scope (account)
appwrite                        | [Error] File: /usr/src/code/app/controllers/general.php
appwrite                        | [Error] Line: 263
appwrite                        | [Error] Timestamp: 2021-09-19T15:12:56+00:00
appwrite                        | [Error] Method: POST
appwrite                        | [Error] URL: /v1/account
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Invalid password: Password must be between 6 and 32 chars and contain ...
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 745
```

## Your Environment

Appwrite 0.10.4 on debian

**Comment by nishihere19** [comment]
Hi! Can I work on this issue?

**Comment by eldadfux** [comment]
This fix is now available as part of Appwrite 0.12 which is now officially released. You can learn more at: https://dev.to/appwrite/its-here-announcing-the-release-of-appwrite-012-5c8b
